src/workerd/server/tests/python/BUILD.bazel

```diff
@@ -30,6 +30,7 @@ py_wd_test("subdirectory")
 
 py_wd_test(
     "sdk",
+    python_flags = ["0.26.0a2"],
```

Comment:
Can you add a comment here explaining why development is disabled?

Reply:
Will do in separate PR